specs/protocol/holocene/system-config.md

```diff
@@ -0,0 +1,39 @@
+# System Config
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [System config contents (version 0)](#system-config-contents-version-0)
+  - [Scalars](#scalars)
+    - [Holocene `scalar`, `overhead` (`uint256,uint256`) change](#holocene-scalar-overhead-uint256uint256-change)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## System config contents (version 0)
+
+### Scalars
+
+In order to allow rollup operators to adjust the new `operatorFeeScalar` and `operatorFeeConstant`, a change to
+the `scalar` and `overhead` encoding is necessary.
+
+#### Holocene `scalar`, `overhead` (`uint256,uint256`) change
+
+After Holocene activation:
+
+- A new version to the `scalar` encoding is added. It is described as follows.
+
+\*Byte ranges are indicated with `[` (inclusive) and `)` (exclusive).
+
+- `0`: scalar-version byte
+- `[1, 32)`: depending scalar-version:
+  - Scalar-version `2`:
+    - `[1, 8)`: padding, must be zero.
```

Comment:
I believe this should be padded to the start of the `operatorFeeScalar`.

i.e.
```suggestion
    - `[1, 12)`: padding, must be zero.
```